dead_mans_switch_server/controllers/main.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    This module copyright (C) 2015 Therp BV (<http://therp.nl>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+import werkzeug
+from openerp import http
+from openerp.http import request
+
+
+class Main(http.Controller):
+    @http.route('/dead_mans_switch/alive', type='json', auth="none")
```

Comment:
@hbrunn Out of curiosity, how do prevent possible DDOS on the server?
Is there a potential risk here? Technically AFAIU the code, knowing the URL is good enough to post
How to prevent it?